### PR TITLE
remove Code Climate from project

### PR DESCRIPTION
**Pull request by dklimpel**
##### Checklist

- [x] `make test-all` (UNIX) passes. CI will also test this
- [ ] unit and/or integration tests are included (if applicable)
- [ ] documentation is changed or added (if applicable)

### Description of change

Follow up to discussion that Code Climate was enabled in #533 and not used anymore.

----
📚 Documentation preview 📚: https://goss--933.org.readthedocs.build/en/933/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,4 @@
 ---
-env:
-  global:
-    - secure: IH/xOtRl9rcek+YTz1s2lv2njdV1gob0AY21/kGhqUBH2sdztQ/02CteGz5E0c+LfqoUYQglLsejVCNLXQJI39CyZRzysB1wlVOz7YIPNaOLATtMGg0/8Mb7dpeqAV1u3agIi87q3ims1EkmGE9UGNj/qL75oTXBiozEfkbcsheH3Ju9CcO0i006UlNqAG4T++l1JZ4kPmr1r5eNjRu52oWQrlxaOYpZJGnT3qjxkvxcnI/Pgiem0+2aZ7Yluu0lA7UCFWyORoDRUz4pQR3uDG4ADPG4WcpJ601LDZyWtS7EOXVBgWZX9vV9ZVx63nK5w0W8NN6PQOuz0niN9iwR+FsShrPIsh6P9wVZfGifwXK4g2FL2XMBRuZCsQfrpRY6JbnuyGeMqN53ta95y0Yv4H65UQ7bulr4Lnic4+W52G/fl1VUYX8Bts+ecUydIu+afcDZ6GkpVLEURjaek99uO5ewHH7lfVAOQv4wUP8GFUyLXJumJZHvMzyiAXsTHdUZdm+Ap8EPzWFYYnMxySU39MEkIvhfG47X6sA7zAppuoNwd8vU4ZaxSdsme2InUSalETGJZ7ohL+M/tkTMyuyq8uLMRuwscNdgcCjGaHM19NQMrv4nsPmmzTeiDRoFk2wad07Mgsjy07esxw52rs3zAsa37+h74GYFWYo523AiYjw=
```
Comment (dklimpel) [inline .travis.yml:4]: Please double check that this token is part of Code Climate.

**Review by petemounce — APPROVED** [review]
Red PR lgtm. Unfortunately I don't know for sure whether the token in .travis.yml is code-climate's.

File: .travis.yml (hunk 1 above)
Comment (aelsabbahy) [reply .travis.yml:4]: the history seems to indicate that it is. If something breaks i'll know to revive it.

Pretty sure travis uses the other token for releases.

**Review by aelsabbahy — APPROVED** [review]
Unfortunately, this was never utilized.. added more noise than not.

**Comment by aelsabbahy** [comment]
Thanks for all the awesome PRs cleaning up CI and others.

**Pull request merged**